modules/core/src/main/java/org/apache/ignite/configuration/IgniteConfiguration.java

```diff
@@ -562,11 +566,24 @@ public IgniteConfiguration(IgniteConfiguration cfg) {
      * @return Optional grid name. Can be {@code null}, which is default grid name, if
      *      non-default grid name was not provided.
```

Comment:
Add the following line to the Java doc right after @return

"@deprecated Use {@link #setInstanceName} instead.

Refer to IgniteConfiguration.setMarshallerCachePoolSize as an example.